courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb

```diff
@@ -4,22 +4,25 @@
    "cell_type": "markdown",
```

Comment:
<p>it seems that 'loss' is not created, so throws an error when trying to print the message. I guess here you want to have an extra line with loss = loss_mse(X, Y, w0, w1) after updating gradients?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>done</p>